src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.epics.js

```diff
@@ -18,16 +22,21 @@ export const scheduleNewEnrollmentEventEpic = (action$: InputObservable) =>
                 stageId,
                 teiId,
                 enrollmentId,
+                eventId,
                 onSaveExternal,
                 onSaveSuccessActionType,
                 onSaveErrorActionType,
             } = action.payload;
 
+            const { events } = store.value;
+            const existingEnrollment = events[eventId]
+            && [statusTypes.SCHEDULE, statusTypes.OVERDUE].includes(events[eventId].status);
 
             const serverData = { events: [{
                 scheduledAt: scheduleDate,
```

Comment:
Hey, 
I am having troubles re-scheduling the event. I get a 500 error that mentions the invalid format of the date.

<img width="685" alt="Screenshot 2022-09-21 at 10 22 05" src="https://user-images.githubusercontent.com/6817073/191456169-f9baf09d-c187-4d13-b4db-7ac911e9ade9.png">


I tried both types of calendars that we currently support. Also when I use the `dd-mm-yyyy` calendar I get an invalid value. What am I doing wrong?

<img width="535" alt="Screenshot 2022-09-21 at 10 21 29" src="https://user-images.githubusercontent.com/6817073/191456349-5da7bdc2-da03-44e9-a1a7-84a126a0ebb8.png">


<img width="421" alt="Screenshot 2022-09-21 at 10 35 31" src="https://user-images.githubusercontent.com/6817073/191458303-a7cea8c2-821a-4ce7-98af-74c334e288b0.png">

Thank you!